src/extensions/default/SVGCodeHints/SVGTags.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
In the spec, attributes are organized into groups such as "conditional processing attributes", "core attributes", "graphical event attributes", "presentation attributes", "xlink attributes", etc.

I think it would reduce a lot of redunancy which would make future maintenance easier if this organization could be captured in the json data. For example, define each of the "attribute groups" as a list of attributes, and then any tag that uses a particular attribute group could simply specify a flag (e.g. `"coreAttrs": true`).

What do you think?